source/server/http/admin.cc

```diff
@@ -330,19 +330,23 @@ std::string AdminImpl::formatTagsForPrometheus(const std::vector<Stats::Tag>& ta
   return StringUtil::join(buf, ",");
 }
 
+std::string AdminImpl::prometheusMetricName(const std::string& extractedName) {
+  return fmt::format("envoy_{0}", sanitizePrometheusName(extractedName));
```

Comment:
Please add a comment for why this is done, and possibly link to the prometheus doc on best practices.